### PR TITLE
Ensure the whole app have only one TextField file

### PR DESCRIPTION
**Pull request by The-indigo**
Ensure the whole app have only one TextField file

https://user-images.githubusercontent.com/48225066/135425091-1bc1988a-5da5-4975-892e-f5bde287a04a.mov

### CODE REVIEW

File: lib/ui/view/edit_profile/edit_profile_view.dart [hunk 1]
```diff
@@ -98,47 +99,126 @@ class Body extends ViewModelWidget<EditProfileViewModel> {
...
+         onChanged: (value) {
                 model.updateString('', '', '', value);
               },
-              decoration: InputDecoration(
-                  labelText: "Phone", helperText: "Enter your phone number"),
+             initialValue: model.userData.phoneNum,
             ),
+
+
+    //  TextFormField(
```
Comment (pauleke65) [inline lib/ui/view/edit_profile/edit_profile_view.dart:204]: delete unused code

File: lib/ui/view/edit_profile/edit_profile_view.dart [hunk 2]
```diff
@@ -98,47 +99,126 @@ class Body extends ViewModelWidget<EditProfileViewModel> {
...
-              decoration: InputDecoration(
-                  labelText: "What I do", helperText: "HNGi9 X I4G"),
+              initialValue: model.userData.status,
+           helperText: 'HNGi9 X I4G'
+              
             ),
-            TextFormField(
-              initialValue: model.userData.phoneNum,
-              onChanged: (value) {
+
```
Comment (pauleke65) [inline lib/ui/view/edit_profile/edit_profile_view.dart:177]: delete unused code, or specify reason for keeping

**Review by pauleke65 — CHANGES_REQUESTED** [review]
Delete unused codes

File: lib/ui/view/edit_profile/edit_profile_view.dart (hunk 1 above)
Comment (The-indigo) [reply lib/ui/view/edit_profile/edit_profile_view.dart:196]: Done deleting unused code

File: lib/ui/view/edit_profile/edit_profile_view.dart (hunk 2 above)
Comment (The-indigo) [reply lib/ui/view/edit_profile/edit_profile_view.dart:177]: Done deleting the unused code

**Comment by The-indigo** [comment]
> Delete unused codes

Done .

**Comment by pauleke65** [comment]
@The-indigo pull from dev upstream and test all the text fields in the app

**Pull request closed without merging**